cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -125,9 +132,16 @@ data ClientCommand
     !FilePath
     !FilePath
   | SubmitTx
-    { stTopology :: TopologyInfo
-    , stTx       :: FilePath
-    , stCommon   :: CommonCLI
+    { stTopology      :: TopologyInfo
+    , stTx            :: FilePath
+    , stCommon        :: CommonCLI
+    }
+  | SpendGenesisUTxO
+    { ctTx            :: FilePath
```

Comment:
These need to be removed because they are partial record accessors.

Reply:
Done!